src/compiler/compiler.rs

```diff
@@ -742,6 +745,10 @@ gcc
                 debug!("Found clang");
                 return Box::new(CCompiler::new(Clang, executable, &pool)
                                 .map(|c| Some(Box::new(c) as Box<Compiler<T>>)));
+            } else if line == "diab" {
```

Comment:
This if/else if chain probably wants to be a match, eh? (I must have written this while I was still fairly new to Rust!)

Reply:
Yes. I just tend to make patches as less invasive as possible for clean diffs in a PR. But in this case...anyway I'll change that.

Reply:
I replaced the `if` `else` stuff with a match.